instances/technique/electronTomography.jsonld

```diff
@@ -4,11 +4,13 @@
   },
   "@id": "https://openminds.ebrains.eu/instances/technique/electronTomography",
   "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
-  "definition": null,
+  "definition": "Electron tomography is a tomographic technique for obtaining 3-dimensional images with transmission electron microscopy.",
```

Comment:
Electron tomography is a microscopy technique that take a series of images of a thick sample at different angles (tilts) so that tomography can be applied to increase the resolution of the ticker sample.